### ISSUE TITLE
How do you load svg-defs now?

### ISSUE THREAD

**Issue by AhmedFat7y** [ahmadalfy/workflow#34]
**Question**

How do you load svg-defs now? It's not mentioned in the workflow. Are you still using jquery.ajax for that?

**Comment by ahmadalfy** [comment]
Nope, we deprecated async loading of SVGs. We used to embed it in the HTML which in turn most of the time significantly increase the number of DOM nodes and worsening page performance.

We used that to support IE (10 I remember?) but right now we dropped it. Currently we use

```html
<svg><use href="full-path-to-sprite.svg#shape-id" /></svg>
```
